usecase/service.go

```diff
@@ -0,0 +1,284 @@
+package usecase
```

Comment:
Maybe, better name for this file - usecases.go? Cause file service.go already exists in service package.